### ISSUE TITLE
User-visible feedback from the python client

### ISSUE THREAD

**Issue by otoomet** [jlebon/textern#24]
Could it be possible to make the python client's more important output (textern.py) visible to the end users?  I struggled >1hr with installation--nothing visible happened when I pressed C+S+E.  It boiled down to _inotify\_simple_ not being installed.

As a side note, should this also be listed as system requirement?

**Comment by jlebon** [comment]
Yeah, I agree feedback could be improved. Even if just to tell folks to check the browser console.

> As a side note, should this also be listed as system requirement?

`inotify_simple` is shipped as a submodule. Are you sure you cloned with `--recurse-submodules` like in the README? (See also #4). I need to make the `native-install` target catch this!

**Comment by otoomet** [comment]
Thanks.  The suggestion to look at console may be enough.  Maybe also ad a FAQ with this solution?

Sorry, forgot about submodules.  I installed it on two computers, on the first one with `--recurse-submodules`, on the other one not.  I was wondering why it did not work on the other otherwise similar system.
